newm-server/src/main/kotlin/io/newm/server/features/distribution/model/AddArtistResponse.kt

```diff
@@ -0,0 +1,16 @@
+package io.newm.server.features.distribution.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class AddArtistResponse(
+    @SerialName("message")
+    val message: String,
+    @SerialName("success")
+    val success: Boolean,
+    @SerialName("artist_id")
+    val artistId: String,
+    @SerialName("artist_name")
+    val artistName: String
+)
```

Comment:
Any possibility of a missing filed on this response? If so, maybe make fields nullable. Same applicable for all other responses that come from the Evara server

Reply:
`artist_name` is a required field in the `AddArtistRequest`. It just gets echoed back here.  I'll be checking all of these once I start making actual queries in some unit tests in the next PR.